apps/docs/content/guides/platform/manage-your-usage/edge-function-invocations.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: 'manage-usage-edge-function-invocations'
+title: 'Manage Edge Function Invocations usage'
+---
+
+## What you are charged for
+You are charged for the number of times your functions get invoked, regardless of the response status code.
+
+## How charges are calculated
+Edge Function Invocations are billed using Package pricing, with each package representing 1 million invocations. If your usage falls between two packages, you are billed for the next whole package.
+
+### Example
+For simplicity, let's assume a package size of 5 and a charge of $2 per package.
```

Comment:
I think we should use the normal package sizes as it will be be easier to understand